pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -66,6 +66,54 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         return {parity_high, parity_middle, parity_low};
     }
 
+    static auto revWireParity(size_t rev_wire0, size_t rev_wire1,
```

Comment:
We can probably define this and the next one as `inline`. Also, worth a try to make it `constexpr` too, (need to check if `fillTrailingOnes` and `fillLeadingOnes` are compatible).

Also, since we know the return type, it would be good to add it as an arrow type: `-> std::array<size_t,parity_size>`

Reply:
Sure!